build.gradle

```diff
@@ -0,0 +1,31 @@
+import edu.wpi.first.toolchain.NativePlatforms
+
+apply plugin: 'cpp'
+apply plugin: 'edu.wpi.first.GradleRIO'
+
+model {
+  components {
+    Sleipnir(NativeLibrarySpec) {
+      targetPlatform NativePlatforms.desktop
+      targetPlatform NativePlatforms.roborio
+
+      sources {
+        sources.cpp {
+          source {
+            srcDir 'src/main/cpp'
+            include '**/*.cpp', '**/*.cc'
```

Comment:
Sleipnir doesn't use .cc files, though its dependencies do. This won't capture them.